### ISSUE TITLE
Saving videos does not refresh gallery (samsung device)

### ISSUE THREAD

**Issue by MoeizN** [BlueBubblesApp/bluebubbles-app#2846]
Using a Samsung Galaxy S23 Ultra

Not sure if this a Samsung problem or a BB problem. App is fully updated and I've reset it a few times.

I'm using a custom folder location (Pictures\BlueBubbles) to download my media. Pictures refresh immediately. Videos and live photos do not update in the gallery until I restart my phone. Even force closing the gallery app doesn't refresh it either. 

Just wanted to let you know

**Comment by rnexusv** [comment]
Facing this same issue on both a Samsung Galaxy S23 Ultra & Samsung Galaxy S25 Ultra

**Comment by jjoelj** [comment]
Does this only happen with .mov files or also .mp4s?

**Comment by rnexusv** [comment]
It only happens with .mov files

**Comment by jjoelj** [comment]
Yeah this isn't really a BlueBubbles issue, though I guess we could add an option to convert movs when saving them

**Comment by rnexusv** [comment]
I figured it wasn't, I was looking around online and it seems to be a samsung gallery app issue, I have found an app that can covert mov to mp4 it's not ideal but it works, if that could be added as an option in the app that would be very nice

**Comment by dhruvb14** [comment]
Same issue on Pixel 9. Someone sent me a video take on iPhone 14 Pro Max with default settings and I can view the video but when I click save I can not see the video in Photos app or via Files app. It's nowhere to be found. 

When clicking information on the video inside BB I see `mime: video MP4`
